### DISCUSSION TITLE
`evil-yank` doesn't invoke `minad/goggles`

### DISCUSSION THREAD

**Discussion by azzamsa** [abougouffa/minemacs#83] [Q&A] [answered]
Hi.

It is defined here:

https://github.com/abougouffa/minemacs/blob/16db93aa9da21c946d83a530fff24219a67ad2ef/modules/me-editor.el#L166

But it doesn't flash during `evil-yank`. However, it flashes during `evil-delete`.

[Screencast from 2023-06-21 12-46-15 (trimmed).webm](https://github.com/abougouffa/minemacs/assets/17734314/d574a345-c18c-48a6-854f-cb3fb5799ee5)

Thanks!

**Comment by abougouffa** [comment, marked as answer]
Yep, I noticed this and I didn't manage to get it to work correctly!

**Reply by azzamsa** [reply]
Should we report upstream?

**Reply by abougouffa** [reply]
I think it is only related to `evil`! Don't know if it is intended for use with evil or not!

**Reply by azzamsa** [reply]
Will explore https://github.com/minad/goggles/issues/8

**Reply by abougouffa** [reply]
If I remember correctly, I've used `evil-goggles` before and didn't get satisfied with the result. It was noticeably slow, and the default colors were too faded for my taste.

**Reply by azzamsa** [reply]
`evil-goggles` works fine for me. Use it for 2 days and I don't notice any slowdown. Maybe you can re-explore it.
